packages/ui/src/components/va-select/VaSelect.demo.vue

```diff
@@ -11,6 +11,107 @@
       />
       <p>Value: {{ defaultSingleSelect.value }}</p>
     </VbCard>
+    <VbCard
+      title="Autocomplete"
+      style="width: 400px;"
+    >
+      <va-select
+        v-model="defaultSingleSelect.value"
+        class="mb-6"
+        placeholder="Start to write..."
+        :options="defaultSingleSelect.options"
+        autocomplete
+        highlight-matched-text
+      />
+      <p>Value: {{ defaultSingleSelect.value }}</p>
+    </VbCard>
+    <VbCard
+      title="Multiple autocomplete"
+      style="width: 400px;"
+    >
+      <va-select
+        v-model="defaultMultiSelect.value"
+        class="mb-6"
+        placeholder="Start to write..."
+        :options="defaultMultiSelect.options"
+        multiple
+        autocomplete
+        highlight-matched-text
+      />
+      <p>Value: {{ defaultMultiSelect.value }}</p>
+    </VbCard>
+    <VbCard
+      title="Multiple autocomplete with max visible options"
+      style="width: 400px;"
+    >
```

Comment:
![image](https://user-images.githubusercontent.com/5394573/217972888-ee07ad0c-6305-40b6-96d9-7009f634e31b.png)

When I type `tw`, then click outside of select - it still retains focus:

![image](https://user-images.githubusercontent.com/5394573/217973157-5bf85864-4f0e-4bb3-9296-441766132f83.png)

I expect to lose focus.

Reply:
It's base VaSelect behaviour: when dropdown closes, input gets the focus. I'll make an exception for autocomplete mode, but still have some doubts about this.

Reply:
> It's base VaSelect behaviour

It was an invalid behaviour then. I checked refs and select always loses focus on click outside.

Reply:
Okey, so i'll make it default for the whole VaSelect, not only for autocomplete.